src/Tests/API/APIApprovals.ApproveAzureServiceBusTransport.approved.txt

```diff
@@ -479,12 +479,22 @@ namespace NServiceBus.AzureServiceBus
         public System.Threading.Tasks.Task Send(Microsoft.ServiceBus.Messaging.BrokeredMessage message) { }
         public System.Threading.Tasks.Task SendBatch(System.Collections.Generic.IEnumerable<Microsoft.ServiceBus.Messaging.BrokeredMessage> messages) { }
     }
-    public class NamespaceInfo
+    public class NamespaceDefinition : System.IEquatable<NServiceBus.AzureServiceBus.NamespaceDefinition>
```

Comment:
We've agreed to go with the following change:
`NamespaceInfo` to be renamed as `RuntimeNamespaceInfo`
and
`NamespaceDefinition` to be renamed as `NamespaceInfo`